wiki/en/en-Adminstration.md

```diff
@@ -5,4 +5,172 @@ lang: "en"
+1. For Pull Requests, additionally do the following:
+   1. Project Board: Move to *Waiting on Team*
+   1. Request Review from two main developers who topic specialists for the PR.
+
+### Reviewing Pull Requests
+1. Add the appropriate Milestone once it is clear that it will go into the next release.
+1. Link any relevant Issues to the PR.
+1. Do a compile test, check for warnings.
+1. Check for correctness, possibly buggy or unstable behavior.
+1. Check for possible performance regressions.
+1. Check coding style.
+1. Check for CI to be green.
+1. If you accept
+1. jamuluswebsite: KB articles: Just check for spam. No deep review required.
+
+### Merging Pull Requests
+
+1. If you are the first Reviewer and you Accept the PR, request review from a second maintainer.
+1. If you are the second Reviewer and you Accept the PR, merge it...
+   - unless you want to request a review from a third Reviewer.
+   - unless this is a PR from a main developer, in which case the author of the PR is supposed to merge themselves.
+1. Add the Milestone (if not done previously)
+1. Merge in a history-friendly way:
+  - Default to Squash merges, especially if the PR consists of multiple individual `fix ...` commits.
+  - Use a regular merge if it makes sense to preserve history and if the individual commits are in a buildable, workable state.
+  - Edit the commit message to be clear in what area of the code it touches. Use imperative. Example: `Update German translation for 3.7.0`. Avoid: `Improving performance`
+
+
+## Discord Chat
+We use the Discord chat for quicker, loose interaction.
+All decision and results should still be documented in permanent Github Issues, Discussions or Comments.
+
+# Decision making
+We use [lazy consensus](https://couchdb.apache.org/bylaws.html#lazy) by default.
+This means that any reversible decision is considered supported by the team as long as nobody objects.
+This also means that everyone must be transparent about their intentions and actions in order for every other member to have a chance to comment.
+
+# Main developers team structure
+Adding or removing team members requires a majority vote by all main developers.
+
+## Accepting new team members
+- Invite to Github org and add permissions <!-- TODO: which/where? -->
+- Add mail address to team@ <!-- TODO: how? -->
+- Invite to Discord server
+- Subscribe to notifications for the jamulussoftware repositories
+
+## Removing team members
+Reverse the onboarding checklist.
+
+# Repositories
+## Jamulus
+## Jamulus website
+## Assets (ASIO4ALL)
+## Install scripts
+
+# Infrastructure
+## Official Directory Servers
+TODO: @pljones'/@corrados' infra
+
+## Update check servers
+TODO: @pljones'/@corrados' infra
+
+## DNS
+TODO: @gilgongo?
+
+## Email
+TODO: @gilgongo?
+
+## Domain
+TODO: Who owns it / has access?
```

Comment:
If you give me your SSH key I can add you to the scp account for DNS changes.

Reply:
I'd rather not like to have access if others already have it for security reasons.

Reply:
Do we have a policy on the minimum number of administrators with access to ensure continuity of service in case of some failure?  The more geographically distributed the better, too.

Reply:
Right now it's just me and @softins but if anyone wants to send me their SSH key they can be added. "The more geographically distributed the better". Is that in case of nuclear attack?